src/components/Maps.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <v-carousel>
+    <v-carousel-item v-for="(item,i) in items" :key="i" :src="item.src"></v-carousel-item>
+  </v-carousel>
+</template>
+
+<script>
+export default {
```

Comment:
Avoid same names in components and views. Also You should explicit define name of your component like:
`export default {
  name: 'ComponentName',
  data() { ... }
}`